Makefile

```
@@ -162,3 +141,15 @@ properclean: IMAGE       ?= $(REGISTRY)/$(ORG)/$(REPO)/$(ENVIRONMENT):$(IMAGE_TA
 properclean:
 	rm -rf $(DISTDIR) $(TESTDIR)
 	$(DOCKER) rmi $(IMAGE)
+
+.PHONY: git2go
+git2go: $(VENDORDIR)/github.com/libgit2/git2go/v31/static-build/install/lib/pkgconfig/libgit2.pc
+
+$(VENDORDIR)/github.com/libgit2/git2go/v31/static-build/install/lib/pkgconfig/libgit2.pc: $(VENDORDIR)/github.com/libgit2/git2go/v31/vendor/libgit2
+	$(MAKE) -C $(VENDORDIR)/github.com/libgit2/git2go/v31 install-static
+
+$(VENDORDIR)/github.com/libgit2/git2go/v31/vendor/libgit2: $(VENDORDIR)/github.com/libgit2/git2go
+	$(GIT) -C $(VENDORDIR)/github.com/libgit2/git2go/v31 submodule update --init --recursive
+
+$(VENDORDIR)/github.com/libgit2/git2go:
+	$(GIT) clone --branch v31.7.9 --recurse-submodules https://github.com/libgit2/git2go.git $@/v31
```

```suggestion
	$(GIT) clone --branch $(GIT2GOBRANCH) --recurse-submodules https://github.com/libgit2/git2go.git $@/v31
```

We should probably make it a variable so it is easier to change/we don't forget about it.

---

Then we need a bit of Makefile magic to strip the `.7.9` out of the `v31.7.9` variable value, probably it is fine to leave as is for now